pucauto.py

```diff
@@ -10,10 +10,21 @@
 from bs4 import BeautifulSoup
 from lib import logger
 
-
-with open("config.json") as config:
-    CONFIG = json.load(config)
-
+try:
+    import config
+except:
+    print 'No config.py found!'
+    exit()
+
+config_vars = dir(config)
+for var in ['username', 'password', 'min_value']:
+    if var not in config_vars:
+        print "Required configuration '%s' not found!" % (var,)
+        exit()
+if 'find_add_ons' not in config_vars:
```

Comment:
I guess this is one thing I dislike about the .py approach vs the .json approach. I can't just call CONFIG.get(...) and if it returns `None` make decisions. Now I have to provide defaults for everything at the top of the script. Hmm.